example/cr/jsonschema-pb-cr.yaml

```diff
@@ -1,9 +1,10 @@
+{{- if .Values.schemaregistry.enabled }}
 apiVersion: "schemaregistry.infoblox.com/v1"
 kind: Jsonschema
 metadata:
-  name: {{ .release.namespace }}.pb.event
+  name: {{ .release.namespace }}-pb-event
```

Comment:
>  It would be ideal to allow users to name these resources ie. {{ include "chartname.fullname" . }}

they can do that, using the `--crnamespace` option. `{{ .release.namespace}}` is just the default

> Are there multiple jsonschemas inside a single service? If so, we'll need some sort of name to append here to differentiate between the various schemas.

that's what the `-pb-event` is coming from. Take a look at the `example/schema/pb` directory - there are five `.jsonschema` files in that directory, and they are all part of this same CR.